src/main/java/com/epam/laboratory/restapipractice/controller/ClientController.java

```diff
@@ -67,21 +64,10 @@ public ResponseEntity read(@PathVariable(name = "id") Long id, Locale locale) {
     @Operation(summary = "Клиенты", description = "Возвращает список клиентов")
     @LogInvocation
     public ResponseEntity<ClientsListResponse> getAllClients() {
-        final List<ClientEntity> clients = clientService.getAllClients();
-        if (clients == null) {
-            return new ResponseEntity<>(HttpStatus.NOT_FOUND);
-        }
-        final ClientsListResponse clientsListResponse = new ClientsListResponse(
-                clients.stream().map(clientEntity -> new ClientResponse(clientEntity.getId(),
-                                clientEntity.getClientName(),
-                                clientEntity.getOrders().stream()
-                                        .map(orderEntity -> new ClientResponse.ClientOrderResponse(orderEntity.getId(),
-                                                orderEntity.getCompleted(),
-                                                orderEntity.getDeliveryInf()))
-                                        .collect(Collectors.toList())))
-                        .collect(Collectors.toList())
-        );
-        return new ResponseEntity<>(clientsListResponse, HttpStatus.OK);
+        final ClientsListResponse clientsListResponse = clientService.getAllClients();
+        return clientsListResponse != null
+                ? new ResponseEntity<>(HttpStatus.OK)
```

Comment:
new ResponseEntity<>(HttpStatus.OK) this will return empty response body, we need ClientsListResponse  in the response body